### ISSUE TITLE
Mock & Write Up the Landing Page Content

### ISSUE THREAD

**Issue by corygalyna** [ZORALab/demo-web-prestige#2]
# Description
```
Please provide a short description of what feature you're looking forward to
add into the existing repository below. Please include the story behind your
idea as well to give a better visualization of your idea.
```
Design the landing page content and sitemaps matching the requirements.

# Expected Behavior
```
Please specify the expected behavior of your requested feature. Some great and
helpful pointers are your expected interface (e.g. command patterns, simple
sketches of the user interface, etc).
```
Draft is out so that Holloway can work it once he finishes everything in time.

# Current Absent Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
Nothing. It's a clean slate.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Cory, please make sure the content is also our pitch. Don't get carried away with those photo albums.

**Comment by corygalyna** [comment]
Ok, landing script just PM-ed you. Please check.

**Comment by hollowaykeanho** [comment]
Just completed reviewing. Yeah.. green light for now.

**Comment by hollowaykeanho** [comment]
Released in version `v1.0.0`.
